### PR TITLE
Fixed issue with app startup. Updated README. Minor refactoring

### PR DESCRIPTION
**Pull request by bkomar**

### CODE REVIEW

File: scripts/start.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# clear all previous docker artifacts
+./$BASEDIR/stop.sh
+
+# creating network to run all related containers in
+docker network create eschool-network
+
+# starting container with mysql DB
+docker run --name eschool-mysql \
+    -p 3606:3606 \
```
Comment (SergiyBojko) [inline scripts/start.sh]: Shouldn't it be -p 3606:3306?

Comment (bkomar) [reply scripts/start.sh]: it is -p 3606:3606. Or i'm missing something?

Comment (bkomar) [reply scripts/start.sh]: got it!

Comment (bkomar) [reply scripts/start.sh]: fixed

**Pull request merged**